src/main/java/com/example/demo/mapper/RoomMapper.java

```diff
@@ -0,0 +1,39 @@
+package com.example.demo.mapper;
+
+import com.example.demo.entity.RoomEntity;
+import com.example.demo.model.RoomModel;
+import org.springframework.stereotype.Component;
+import org.modelmapper.ModelMapper;
+
+import java.util.Optional;
+
+@Component
+public class RoomMapper implements BasicMapper<RoomEntity, RoomModel> {
```

Comment:
potrzebujemy do każdego obiektu tych mapperów?

Reply:
Też na to patrzyłem, że ta biblioteka produkuje dużo biolderplate'u i nie wiemy jak działa pod spodem. Nie czytelniej byłoby mapować samemu do DTOsów?

Ale plus za odróżnienie modelu od obiektów DTO dla odpowiedzi.